README.md

```diff
@@ -41,6 +41,13 @@ Default: `false`
 
 If true, will throw an error if any warnings or errors were found.
 
+### breakOnWarning
+
+Type: `boolean`  
+Default: `false`
+
+If true, will stop rollup if any warnings or error were found without adding error stack trace. This helps to have clean warnings and errors from eslint without our added error stack trace.
```

Comment:
@TrySound IMHO it should be `warnings and errors` opposed to `warning and error`, as there may be multiple of them.